parent/pom.xml

```diff
@@ -208,9 +208,9 @@
                 <version>${swagger.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.eclipse.jetty.orbit</groupId>
-                <artifactId>javax.servlet</artifactId>
-                <version>${jetty-orbit-javax-servlet.version}</version>
+                <groupId>javax.servlet</groupId>
+                <artifactId>javax.servlet-api</artifactId>
+                <version>${javax-servlet.version}</version>
```

Comment:
You left the dependency only in the `dependencyManagement` section in the parent pom, which doesn't actually pull it as a dependency, just declares which version to use in case anyone needs it. If classes from it are still directly referenced, then put it explicitly in the `dependencies` sections in modules that need it. It works now because it's a transitive dependency of jetty.